src/DotNetLightning.Core/Serialize/Features.fs

```diff
@@ -194,9 +199,7 @@ type FeatureBit private (bitArray) =
             else
                 return (FeatureBit(ba))
         }
-    static member Zero =
-        let b: bool array = [||]
-        b |> BitArray |> FeatureBit
+    static member Zero = FeatureBit()
```

Comment:
is this change really needed for this PR?

Reply:
Nope. I can remove it.